### PR TITLE
chore: query range v3 metrics use v4 tables

### PR DESCRIPTION
**Pull request by srikanthccv**
### Summary

Part of https://github.com/SigNoz/signoz/issues/4845

#### Screenshots

NA

### CODE REVIEW

**Comment by nityanandagohain** [comment]
can you give me some idea when is time_series_v4 , time_series_v4_6hrs, time_series_v4_1day get used w.r.t to different queries ?

**Comment by nityanandagohain** [comment]
> We dynamically adjust what table to use depending on the time range of the query. for range [0-6hrs] : time_series_v4, [6hrs-24hrs]: time_series_v4_6hrs and [24hrs-]: time_series_v4_1day. All of this is done to reduce the amount of data we read.

where is this logic present, or is it yet to be implemented ?

**Comment by srikanthccv** [comment]
It is already part of the codebase https://github.com/SigNoz/signoz/blob/12be6ce02070f8670c8591e5219eb6e9d09ba11e/pkg/query-service/app/metrics/v4/helpers/sub_query.go#L18-L50.

File: pkg/query-service/app/clickhouseReader/reader.go [hunk 1]
```diff
@@ -72,11 +72,17 @@ const (
...
 	signozMetricDBName         = "signoz_metrics"
-	signozSampleLocalTableName = "samples_v2"
-	signozSampleTableName      = "distributed_samples_v2"
-	signozTSTableName          = "distributed_time_series_v2"
-	signozTSTableNameV4        = "distributed_time_series_v4"
-	signozTSTableNameV41Day    = "distributed_time_series_v4_1day"
+	signozSampleLocalTableName = "samples_v4"
+	signozSampleTableName      = "distributed_samples_v4"
+
+	signozTSLocalTableNameV4 = "time_series_v4"
```
Comment (nityanandagohain) [inline pkg/query-service/app/clickhouseReader/reader.go:78]: can you use SIGNOZ_TIMESERIES_v4_6HRS_LOCAL_TABLENAME from constants ?

File: pkg/query-service/app/clickhouseReader/reader.go [hunk 2]
```diff
@@ -72,11 +72,17 @@ const (
...
+	signozTSTableNameV4      = "distributed_time_series_v4"
+
+	signozTSLocalTableNameV46Hrs = "time_series_v4_6hrs"
+	signozTSTableNameV46Hrs      = "distributed_time_series_v4_6hrs"
+
+	signozTSLocalTableNameV41Day = "time_series_v4_1day"
```
Comment (nityanandagohain) [inline pkg/query-service/app/clickhouseReader/reader.go:84]: SIGNOZ_TIMESERIES_v4_1DAY_LOCAL_TABLENAME

**Review by nityanandagohain — APPROVED** [review]

File: pkg/query-service/app/clickhouseReader/reader.go (hunk 1 above)
Comment (srikanthccv) [reply pkg/query-service/app/clickhouseReader/reader.go:78]: I will do another PR.

**Pull request merged**